.github/ISSUE_TEMPLATE/toil.yaml

```diff
@@ -12,7 +12,7 @@ body:
     label: Maintenance type
     options:
       - [Terraform upgrades Apps](https://dev.azure.com/brreg/altinn-apps-ops/_git/tjenesteeier-template?path=/README.md&_a=preview)
-      - [Terraform upgrades Platform]()
```

Comment:
We have documentation for apps too.
https://pedia.altinn.cloud/altinn-3/ops/patching/terraform/apps/

Reply:
changed the link to pedia. When we get confluence pages up and running we should probably change these again